### ISSUE TITLE
Can we use soft link or other to workaround hard link in build-tools/lib/release.sh

### ISSUE THREAD

**Issue by resouer** [kubernetes/kubernetes#36731]
I once encountered this problem before, and would like to discuss it again.

It's about: https://github.com/kubernetes/kubernetes/blob/master/build-tools/lib/release.sh#L242

The problem is: hard link does not work well on shared file system in virtualbox, so this line is 'painful' in vagrant+virtualbox based dev environment, for example, if you want to run `go run hack/e2e.go -v --build`.

But I cannot replace it to soft link since it will break image build later, see: https://github.com/kubernetes/kubernetes/pull/33082

I did not dig this deeply, just fired this issue to discuss it. Can we use soft link or other to workaround hard link in `build-tools/lib/release.sh`?

**Comment by resouer** [comment]
ping @mikedanese any idea to solve/workaround this problem?

**Comment by mikedanese** [comment]
Can you detect the shared filesystem and just do a cp -r?

**Comment by resouer** [comment]
Yes I think so. If there's no other way to improve this from `release.sh`'s side :)
